From 81953ee9438feb5bfdf8a3095f035081798ac468 Mon Sep 17 00:00:00 2001
From: "Simental Magana, Marcos" <marcos.simental.magana@intel.com>
Date: Wed, 7 Feb 2018 11:15:21 -0600
Subject: [PATCH] no-vala-please

---
 configure | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configure b/configure
index 8894299..5d1fd09 100755
--- a/configure
+++ b/configure
@@ -16166,7 +16166,7 @@ if test \
         ! -e ${sd}folks/libfolks_la_vala.stamp \
       ; then :
 
-        enable_vala=yes
+        enable_vala=no
         echo "Vala source needs to be re-compiled; requiring Vala..."
 
 fi
-- 
2.16.1

